windows-release/stage-layout-embed.yml

```
@@ -46,6 +46,23 @@ jobs:
       --preset-embed
     displayName: 'Generate embeddable layout'
 
+  - powershell: >
+      & "$(Python)"
+      "$(Build.SourcesDirectory)\sbom.py"
```

Okay, so I guess we don't have this file without doing a `checkout: self` step first, which is going to mess with every other step and template (because we currently don't checkout multiple repositories anywhere, so they aren't "aware" of it).

It might be easiest to add a custom `git clone` command to clone our own repo into its own directory. I _think_:

```
git clone $(Build.Repository.Uri) -b $(Build.SourceBranch) --single-branch --no-checkout "$(Pipeline.Workspace)\release-tools"
git -C "$(Pipeline.Workspace)\release-tools" checkout $(Build.SourceVersion)
```

And then:
```suggestion
      "$(Pipeline.Workspace)\release-tools\sbom.py"
```

---

Got it! Done in https://github.com/python/release-tools/pull/100/commits/04ee7a65197cdb2b6a884c66283f18cea7ea3851